web/src/components/apps/AppConfig.jsx

```diff
@@ -431,6 +431,13 @@ class AppConfig extends Component {
     this.setState({ displayErrorModal: !this.state.displayErrorModal });
   }
 
+  navigateToUpdatedConfig = (app) => {
+    this.setState({ showNextStepModal: false });
+
+    const pendingVersions = app?.downstreams[0]?.pendingVersions;
```

Comment:
how could you guarantee that pendingVersions include the latest version?

Reply:
because after you save the changes the new version is created and that's added to the top of the downstreams pending versions.

Reply:
yeah, but that's coming from props, and it's a separate api request that updates the app, the downstream, and the pending versions, so there could be a delay between when the modal shows up, and the pending versions being updated, right?

Reply:
handler to save config is updating the app downstreams. modals shows after changes are saved, not while changes are saving.